src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining/InnerApi/Responses/CouseLocationModel.cs

```diff
@@ -13,6 +14,7 @@ public class CouseLocationModel
         public double? Longitude { get; set; }
     }
 
+    //MFCMFC this is probably going
```

Comment:
I presume these are notes for the next work, make sure none of these are left behind. Also suggest make notes out of the code.